subprojects/ear/build.gradle.kts

```diff
@@ -37,6 +37,7 @@ dependencies {
 
 strictCompile {
     ignoreRawTypes() // raw types used in public API
+    ignoreDeprecations()
```

Comment:
This will hide any usage of deprecated members in the whole project. I would have added warning suppression annotation in places where deprecated members are in use instead.